packages/web3-hooks/base/src/useHiddenAddressConfig.ts

```diff
@@ -0,0 +1,60 @@
+import { EMPTY_LIST, EMPTY_OBJECT, NextIDPlatform, type PluginID } from '@masknet/shared-base'
+import { useQuery } from '@tanstack/react-query'
+import { useWeb3State } from './useWeb3State.js'
+
+type Result = Record<string, string[]> | string[]
+type StorageValue =
+    | string[]
+    | { hiddenAddresses?: string[] }
+    | {
+          hiddenAddresses?: Result
+      }
+/**
+ *
+ * Get unlisted address by persona pubkey.
+ *
+ * @param personaPubkey The persona identifier in hex string
+ * @param pluginID The plugin id as store key
+ * @returns
+ */
+export function useHiddenAddressConfig(personaPubkey?: string, pluginID?: PluginID) {
```

Comment:
Recorder the parameters from `[pluginId, pubkey]` to `[pubkey, pluginId]` according to storage fields order.